examples/transactions.py

```diff
@@ -128,7 +128,9 @@ def transfer(
     )
 
     transfer_message = cryptocurrency_module.Transfer()
-    transfer_message.to.CopyFrom(types_module.PublicKey(data=to_key.value))
+    public_key = types_module.PublicKey(data=to_key.value)
+    caller_address = message_generator.pk_to_caller_address(public_key)
```

Comment:
Nit: It's not a caller address, but rather a callee one. Thus, I'd suggest to rename method `pk_to_address` as well to avoid confusion.

It would be nice to have a short comment explaining what are we doing here, since it's an example, e.g.:

```python
# Recipient address can be derived from its public key.
```